### PR TITLE
feat(client): ✨ Add Credits 

### LINKED ISSUE #190: Credits
Credits should be listed below the rules (displayed after hitting the rules button). These credits should list the team members, the artist, and have link to the main github page and also a link to the create a new issues page.

### PR DESCRIPTION
**Pull request by njumaah**

### CODE REVIEW

File: client/src/routes/+layout.svelte [hunk 1]
```diff
@@ -155,6 +155,21 @@
...
+    <a href="https://github.com/SBAsai" target="_blank">Shunsuke Asai - Team Member</a> <br />
+    <a href="https://github.com/zaverbac" target="_blank">Zac Averbach - Team Member</a> <br />
+    <a href="https://github.com/gchamb" target="_blank">Giantte Chambers - Team Member</a> <br />
+    <a href="https://github.com/cmcclusky" target="_blank">Cameron McClusky - Team Member</a> <br />
+    <a href="https://github.com/mrjoshua520" target="_blank">Josh Salkeld - Team Member</a> <br />
+    <a href="https://twitter.com/00_yupia" target="_blank">yokaih - Artist</a> <br />
+
+    <h3 class="mdc-typography--headline5">Links</h3>
+    <a href="https://github.com/cat-acronym-team/cat-me-if-you-can" target="_blank">Github</a> <br />
+    <a href="https://github.com/cat-acronym-team/cat-me-if-you-can/issues/new" target="_blank">Have an issue?</a>
```
Comment (nstringham) [inline client/src/routes/+layout.svelte]: All these link should have noreferrer
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer

File: client/src/routes/+layout.svelte [hunk 2]
```diff
@@ -155,6 +155,21 @@
       <li>Scenario 1 will result in the cats winning. The app is now fish-free!</li>
       <li>Scenario 2 will result in the catfish winning and cats dropping lower in the food chain…</li>
     </ul>
+
+    <h3 class="mdc-typography--headline5">Credits</h3>
+    <a href="https://github.com/nstringham" target="_blank">Nate Stringham - Team Lead</a> <br />
+    <a href="https://github.com/njumaah" target="_blank">Nathan Jumaah - Deputy Team Lead</a> <br />
```
Comment (nstringham) [inline client/src/routes/+layout.svelte]: Personally I would probably just say "Team Member" for Nathan but I'm fine with whatever

File: client/src/routes/+layout.svelte [hunk 3]
```diff
@@ -177,6 +192,11 @@
     left: 8px;
   }
 
+  h3 {
+    margin-top: 10px;
+    margin-bottom: 10px;
```
Comment (nstringham) [inline client/src/routes/+layout.svelte]: These can be combined into margin-block
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block

**Review by nstringham — DISMISSED** [review]

File: client/src/routes/+layout.svelte [hunk 4]
```diff
@@ -155,6 +155,21 @@
...
+    <h3 class="mdc-typography--headline5">Credits</h3>
+    <a href="https://github.com/nstringham" target="_blank">Nate Stringham - Team Lead</a> <br />
+    <a href="https://github.com/njumaah" target="_blank">Nathan Jumaah - Deputy Team Lead</a> <br />
+    <a href="https://github.com/Muaad2001" target="_blank">Muaad Abdulaziz - Team Member</a> <br />
+    <a href="https://github.com/SBAsai" target="_blank">Shunsuke Asai - Team Member</a> <br />
+    <a href="https://github.com/zaverbac" target="_blank">Zac Averbach - Team Member</a> <br />
+    <a href="https://github.com/gchamb" target="_blank">Giantte Chambers - Team Member</a> <br />
+    <a href="https://github.com/cmcclusky" target="_blank">Cameron McClusky - Team Member</a> <br />
+    <a href="https://github.com/mrjoshua520" target="_blank">Josh Salkeld - Team Member</a> <br />
+    <a href="https://twitter.com/00_yupia" target="_blank">yokaih - Artist</a> <br />
```
Comment (nstringham) [inline client/src/routes/+layout.svelte]: Should the first letter of her name be capitalized?

File: client/src/routes/+layout.svelte (hunk 2 above)
Comment (njumaah) [reply client/src/routes/+layout.svelte]: This was my position, so I'm going to keep it that way.

File: client/src/routes/+layout.svelte (hunk 1 above)
Comment (njumaah) [reply client/src/routes/+layout.svelte]: Done

File: client/src/routes/+layout.svelte (hunk 3 above)
Comment (njumaah) [reply client/src/routes/+layout.svelte]: Done

**Review by mrjoshua520 — DISMISSED** [review]

File: client/src/routes/+layout.svelte [hunk 5]
```diff
@@ -177,6 +194,10 @@
     left: 8px;
   }
 
+  h3 {
+    margin-block: 10px;
```
Comment (nstringham) [inline client/src/routes/+layout.svelte]: this feels a bit cramped on large screens.

we tried only changing the `margin-bottom` which looked kinda good (Josh and Giantte liked this the best)

we also considered using the compact styles only on narrow devices

I think more discussion is needed

File: client/src/routes/+layout.svelte [hunk 6]
```diff
@@ -155,6 +155,23 @@
...
+    <h3 class="mdc-typography--headline5">Credits</h3>
+    <a href="https://github.com/nstringham" rel="noreferrer" target="_blank">Nate Stringham - Team Lead</a> <br />
+    <a href="https://github.com/njumaah" rel="noreferrer" target="_blank">Nathan Jumaah - Deputy Team Lead</a> <br />
+    <a href="https://github.com/Muaad2001" rel="noreferrer" target="_blank">Muaad Abdulaziz - Team Member</a> <br />
+    <a href="https://github.com/SBAsai" rel="noreferrer" target="_blank">Shunsuke Asai - Team Member</a> <br />
+    <a href="https://github.com/zaverbac" rel="noreferrer" target="_blank">Zac Averbach - Team Member</a> <br />
+    <a href="https://github.com/gchamb" rel="noreferrer" target="_blank">Giantte Chambers - Team Member</a> <br />
+    <a href="https://github.com/cmcclusky" rel="noreferrer" target="_blank">Cameron McClusky - Team Member</a> <br />
+    <a href="https://github.com/mrjoshua520" rel="noreferrer" target="_blank">Josh Salkeld - Team Member</a> <br />
+    <a href="https://twitter.com/00_yupia" rel="noreferrer" target="_blank">Yokaih - Artist</a> <br />
```
Comment (nstringham) [inline client/src/routes/+layout.svelte:168]: if we capitalize this we should do it on the readme

**Review by nstringham — DISMISSED** [review]

File: client/src/routes/+layout.svelte (hunk 6 above)
Comment (njumaah) [reply client/src/routes/+layout.svelte:168]: see #197

File: client/src/routes/+layout.svelte (hunk 5 above)
Comment (njumaah) [reply client/src/routes/+layout.svelte]: after playing around with both, i prefer the margin-bottom as well.

**Review by nstringham — CHANGES_REQUESTED** [review]
It doesn't look to me like you updated the readme

**Review by nstringham — APPROVED** [review]

**Pull request merged**